src/transformState.js

```diff
@@ -4,8 +4,27 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
-  // write code here
+function transformState(state, ...actions) {
```

Comment:
```suggestion
function transformState(state, actions) {
```
why do you need this?

Reply:
I accidentally add it, and now after I deleted it my code go wrong.